doc/news/_preparation_next_release.md

```diff
@@ -34,11 +34,34 @@ docker run -it elektra/elektra
 ## Highlights
 
 - `kdb` now prohibits write operations on cascading keys that miss a corresponding existing key. See the details in the `Tools` section below and the new subsection on cascading writes in the [tutorial](/doc/tutorials/cascading.md) on cascading keys for further information. _(Alexander Firbas)_
-- <<HIGHLIGHT1>>
 - <<HIGHLIGHT2>>
 - <<HIGHLIGHT3>>
 
-### <<HIGHLIGHT1>>
+### Redshift and Elektra
+
+We have created a version of [Redshift](https://github.com/jonls/redshift/) that uses Elektra for configuration management!
+
+Redshift's code for loading configuration files, parsing CLI options and validating configuration was refactored to use Elektra.
```

Comment:
Verbs should be early and also state our goal: that apps have less code for config stuff.

```suggestion
We removed and refactored Redshift's code for loading configuration files, parsing CLI options and validating configuration to use Elektra.
```

Reply:
Thanks, I've adopted and extended your suggestion!